src/lib/load_tectonicsummary.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+// ----------------------------------------------------------------------
+// Tectonic Summary data download/uncompress
+// ----------------------------------------------------------------------
+
+$answer = promptYesNo(
+    "\nUpdating Tectonic Summary dataset. Existing data will be deleted, " .
+    " continue?",
+    true
+  );
+
+if (!$answer) {
+  echo "Skipping tectonic summary.\n";
+  return;
+}
+
+
+$tectonicSql = configure('TECTONIC_SQL',
+    $defaultScriptDir . DIRECTORY_SEPARATOR . 'tectonicsummary.sql',
+    "Tectonic summary schema script");
+$dbInstaller->runScript($tectonicSql);
+echo "Success!!\n";
+
+
+// download tectonic summary data
+echo "\nDownloading and loading tectonic summary data:\n";
+$url = configure('GLOBAL_TECTONIC_SUMMARY_URL',
+    'ftp://hazards.cr.usgs.gov/web/hazdev-geoserve-ws/tectonic/',
+    "Tectonic summary download url");
+$filenames = array('tectonicsummary.zip');
```

Comment:
download the individual files, instead of the zip, so it's easier to update an individual file.